src/components/Header.js

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useThemeContext, useThemeToggleContext } from '../providers/ThemeProvider';
+
+const Header = () => {
+  const setTheme = useThemeToggleContext();
+  const theme = useThemeContext();
+
+  const handleChange = () => {
+    if (theme === 'light') {
+      setTheme('dark');
+    } else {
+      setTheme('light');
+    }
+  };
+  console.log(theme);
```

Comment:
Dear Dantacho,

plz can you remove this console.log?

Reply:
As I see, the commit was old, but now appears to me a new branch with the code deleted. Thanks Dante!